css/portfolio.css

```diff
@@ -0,0 +1,92 @@
+* {
+    box-sizing: border-box;
+}
+/*this overides padding default*/
+
+body {
+    font-size: 62.5%;
+    font-family: "amatic sc", cursive;
+}
+/*this scales fonts when zoom is altered in OS*/
+
+nav {
+    background-color: #DCDCDC;
+    height: 100px;
+    padding-top: 20px;
+    color: white;
+    font-weight: bold;
+    position: fixed;
+    width: 100%;
+
+}
+.menu {
+}
```

Comment:
`letter-spacing: 5px;` this is a way to make space between your letters, might help to make your buttons more readable even when you decrease the size